middlewares/saveBackend.go

```diff
@@ -0,0 +1,24 @@
+package middlewares
+
+/*
+Middleware saveBackend sends the backend name to the logger.
```

Comment:
Same here, this comment should be on the Logger struct (to appear correctly in `godoc`).